test/e2e_node/services/kubelet.go

```diff
@@ -194,6 +194,10 @@ func (e *E2EServices) startKubelet() (*server, error) {
 		unitName := fmt.Sprintf("kubelet-%s.service", unitTimestamp)
 		cmdArgs = append(cmdArgs,
 			systemdRun,
+			"-p", "Delegate=true",
```

Comment:
thanks for updating this to ensure its explicit, i think we had previously assumed this was done globally via systemd config.